src/import/sst_service.rs

```diff
@@ -671,17 +706,40 @@ impl<E: Engine> ImportSst for ImportSstService<E> {
                 CONFIG_ROCKSDB_GAUGE.with_label_values(&[cf, name]).set(v);
             }
 
-            if let LocalTablets::Singleton(tablet) = &self.tablets {
-                match req.get_mode() {
+            match &self.tablets {
+                LocalTablets::Singleton(tablet) => match req.get_mode() {
                     SwitchMode::Normal => self.importer.enter_normal_mode(tablet.clone(), mf),
                     SwitchMode::Import => self.importer.enter_import_mode(tablet.clone(), mf),
+                },
+                LocalTablets::Registry(_) => {
+                    if req.get_mode() == SwitchMode::Import {
+                        if !self.importer.region_import_mode() {
```

Comment:
We should support client to call multiple enter import model with different key ranges.  So here when region_import_mode is true, it should continue to update the import mode regions by the key ranges.